release-1.6/release-notes-draft.md

```diff
@@ -58,12 +58,59 @@
 
 ## Cluster Lifecycle
 ### New Features
+- kubeadm is a simple tool for creating Kubernetes clusters on existing
+  machines. kubeadm's command line interface is now in beta.  Using it is
```

Comment:
Is the entirety of the command line interface in beta? Mention `alpha` subcommand here.